### DISCUSSION TITLE
Retrieving Criteria Proofs when Fulfilling an Order

### DISCUSSION THREAD

**Discussion by warneracw21** [ProjectOpenSea/seaport-js#81] [Q&A] [answered]
When attempting to fulfill an order using Seaport, it is apparent that the **offerCriteria** and **considerationCriteria**  are not provided with the offer's retrieved from the REST endpoint.

Is there a way to calculate these "proofs"?

Specifically referencing  [./src/utils/criteria.js [42]](https://github.com/ProjectOpenSea/seaport-js/blob/21aeed14a3f479269113ea5fcbc11fc08d0eaf3e/src/utils/criteria.ts#L42)
```js
const mapCriteriaItemsToResolver = (criteriaItems, criterias) =>
    criteriaItems.map(({ orderIndex, item, index, side }, i) => {
	const merkleRoot = item.identifierOrCriteria || "0";
	const inputCriteria = criterias[orderIndex][i];

	return {
		orderIndex,
		index,
		side,
		identifier: inputCriteria.identifier,
		criteriaProof: (merkleRoot === "0") ? [] : inputCriteria.proof
	};
});

**Comment by sunwrobert** [comment, marked as answer]
This is a duplicate of https://github.com/ProjectOpenSea/seaport-js/discussions/77
